deployment/centos/deploy.sh

```diff
@@ -1,14 +1,21 @@
-#!/bin/sh
+#!/bin/bash -e
+
+# change this to the current semver tag, e.g. v1.8.0
+GIT_TAG=master
+
+echo "using git tag or branch: ${GIT_TAG}"
+echo
 
 cd ~
 rm -rf scoreboard
 git clone https://github.com/developmentseed/scoreboard.git
 cd scoreboard
-git checkout master
+git checkout ${GIT_TAG}
 
 # installation
 npm install
```

Comment:
Should we use yarn?

Reply:
@kamicut good catch. Because we have `yarn.lock` in the repository, we won't get it's benefits by using `npm` to install javascript dependencies.

A side effect is yarn will need to be added to the deployment docs I am working on. 

will make these changes 👍